cmd/ghemoji/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/donatj/ghemoji"
+)
+
+func main() {
+	// right now, we are not using flag, but we could
+	// so let's plan it
+	flag.Usage = usage
+
+	input, err := readInput()
+	if err != nil {
+		flag.Usage()
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(2)
+	}
+
+	if input == "" {
+		flag.Usage()
+		fmt.Fprintln(os.Stderr, "no input provided")
+		os.Exit(2)
+	}
```

Comment:
```suggestion
```

I might be inclined to omit this check. If I was using this in a script, I'd expect empty string to just silently return an empty string, rather than bark.